presto-docs/src/main/sphinx/release/release-0.261.rst

```diff
@@ -0,0 +1,50 @@
+=============
+Release 0.261
+=============
+
+**Highlights**
+==============
+* Improved performance of sum and average aggregations on decimals.
+* Support for LocationProvider supplied paths in Iceberg connector.
+* Column comments now available in JDBC connector.
+* Hidden columns ``file_modified_time`` and ``file_size`` added to Hive connector.
+* Partition lease support added to Hive connector.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix the Aggregation IF to FILTER rule by not unwrapping the IF expression inside the Aggregation. The rule now rewrites ``AGG(IF(condition, expr))`` to ``AGG(IF(condition, expr)) FILTER (WHERE condition)``.
+* Fix marking coordinator active so it is only done when the required minimum resource managers and required minimum coordinators are available in a cluster. Required resource managers can be configured by ``cluster.required-resource-managers-active`` (default value: 1, disaggregated coordinator only). Required coordinators can be configured by ``cluster.required-coordinators-active`` (default value: 1).
+* Improve performance of sum and average aggregations on decimals.
+* Add additional runtime stats for ``ScanFilterAndProject`` operator. Previously only ``TableScanOperator`` was exposed.
+* Disable ``optimizer.aggregation-if-to-filter-rewrite-enabled`` by default.
+
+SPI Changes
+___________
+* Add ``Identity`` field to ``checkCanSetUser`` for finer granularity of system access control.
+
+Hive Changes
+____________
+* Add hidden column ``$file_modified_time``.
+* Add hidden column ``$file_size``.
+* Add support for partition leases. Configuration property ``hive.partition-lease-duration`` and session property ``partition_lease_duration`` can be used to specify the duration of a lease (e.g. "4h") or disable partition leases ("0h").
+* Add CPU usage of file metadata reading to runtime statistics.
```

Comment:
This is also internal and doesn't have to be included in release notes.

Reply:
These counters are shown in the coordinator web UI and in the return info of the presto-cli in debug mode. So users can see them. Are these considered user-facing?

Reply:
I don't have strong opinions. Generally speaking we try to describe what the feature mean to users in release notes (vs. what's implemented). So if you think this is valuable to users, modify the description to reflect that, e.g. something like "add CPU usage stats to coordinator web UI. The stats can be found at ...".

Reply:
How about
```
 Add CPU usage statistics for file metadata reading to RuntimeStats table in the coordinator UI and debug mode of presto-cli.
```